src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml

```diff
@@ -298,70 +298,70 @@
     <sys:String x:Key="Activate_KMS_Reset.Description">KMS sunucu adını kaldırın ve KMS bağlantı noktasını varsayılan olarak ayarlayın.</sys:String>
 
     <!-- Licenses list -->
-    <sys:String x:Key="Activate_Licenses">Office licenses</sys:String>
-    <sys:String x:Key="Activate_License">Licenses list</sys:String>
-    <sys:String x:Key="Activate_License.Description">Manage installed licenses.</sys:String>
-    <sys:String x:Key="Activate_License_Refresh">_Refresh</sys:String>
-    <sys:String x:Key="Activate_License_Expand">_Expand all</sys:String>
-    <sys:String x:Key="Activate_License_Collapse">_Collapse all</sys:String>
-    <sys:String x:Key="Activate_License_Clear">C_lear list</sys:String>
-    <sys:String x:Key="Activate_License_Status.Unactivated">Unactivated</sys:String>
-    <sys:String x:Key="Activate_License_Status.Activated">Activated</sys:String>
+    <sys:String x:Key="Activate_Licenses">Ofis lisansları</sys:String>
+    <sys:String x:Key="Activate_License">Lisans listesi</sys:String>
+    <sys:String x:Key="Activate_License.Description">Kurulu lisansları yönetin.</sys:String>
+    <sys:String x:Key="Activate_License_Refresh">_Yenile</sys:String>
+    <sys:String x:Key="Activate_License_Expand">_Hepsini genişlet</sys:String>
+    <sys:String x:Key="Activate_License_Collapse">_Hepsini Daralt</sys:String>
```

Comment:
The access key is same as previous one, please select others one. If you don't know the access key, please visit: https://wpf-tutorial.com/control-concepts/access-keys/